mysql-test/suite/rocksdb/t/use_direct_reads_writes.test

```diff
@@ -36,12 +36,12 @@ shutdown_server 10;
 
 --exec echo "" >$MYSQLTEST_VARDIR/log/mysqld.1.err
 
---exec echo "restart:--rocksdb_use_direct_writes=1 --rocksdb_allow_mmap_writes=1" >$_expect_file_name
+--exec echo "restart:--rocksdb_use_direct_io_for_flush_and_compaction=1 --rocksdb_allow_mmap_writes=1" >$_expect_file_name
```

Comment:
Not in the scope of this PR, but this whole testcase must be cleaned up WRT error log handling